keps/sig-api-machinery/4008-crd-ratcheting/README.md

```diff
@@ -923,13 +1099,15 @@ feature flags will be enabled on some API servers and not others during the
 rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
+This feature will not impact rollouts or already-running workloads.
 
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
+If `apiextensions_apiserver_update_ratcheting_time` is taking a long time.
```

Comment:
Can we characterize "long" more explicitly? Order of magnitude, maybe? 1ms? 1s? 1m?

Reply:
updated to say that "long" is in the order of 100ms